android/src/main/java/com/appcues/sdk/capacitor/AppcuesPluginConfig.kt

```diff
@@ -23,6 +23,8 @@ class AppcuesPluginConfig(call: PluginCall) {
         sessionTimeout?.let { appcuesConfig.sessionTimeout = it }
         activityStorageMaxSize?.let { appcuesConfig.activityStorageMaxSize = it }
         activityStorageMaxAge?.let { appcuesConfig.activityStorageMaxAge = it }
+
+        appcuesConfig.additionalAutoProperties = mapOf("_applicationFramework" to "capacitor")
```

Comment:
This is done in both iOS and Android plugin implementations, since the capacitor plugin structure does not allow for adding supplemental info in the wrapper layer calls. Also, I have not yet found any way to get the capacitor runtime version programmatically. So no `_applicationFrameworkVersion` is included.